localstack/services/install.py

```diff
@@ -405,7 +411,10 @@ def install_local_kms():
 def install_stepfunctions_local():
     if not os.path.exists(INSTALL_PATH_STEPFUNCTIONS_JAR):
         # pull the JAR file from the Docker image, which is more up-to-date than the downloadable JAR file
-        # TODO: works only when running on the host, outside of Docker -> add a fallback if running in Docker?
+        if not has_docker():
+            # TODO: works only when running on the host, outside of Docker -> add a fallback if running in Docker?
```

Comment:
This comment seems incorrect now. It should work as long as we are having access to a docker socket.
```suggestion
            # TODO: works only when a docker socket is available -> add a fallback if running without Docker?
```

Reply:
Great point! 👍